generated/package.json

```diff
@@ -3,11 +3,11 @@
   "sdk-type": "mgmt",
   "author": "Microsoft Corporation",
   "description": "A generated SDK for ContextAPI.",
-  "version": "1.0.0-beta.1",
+  "version": "2.0.0",
```

Comment:
Do we need a major version upgrade? I'd expect that 2.x.x would communicate with api v2, there's also no breaking changes

Reply:
Alex suggested this to make sure that the parent package in production doesn't accidentally use the new child package due to the use of ^ etc